kmod/core/core.c

```diff
@@ -742,7 +705,7 @@ static int kpatch_write_relocations(struct kpatch_module *kpmod,
 			break;
 		case R_X86_64_64:
 			loc = dynrela->dest;
```

Comment:
This part of the patch is unrelated. Looks like an artefact from rebase or something.
Please rebase the patch on top of the current master branch in dynup/kpatch.

Reply:
Yes, maybe I got an old version. I will rebase the patch on top of the current master branch in dynup/kpatch.